src/app/authorization/login/login.component.css

```diff
@@ -1,13 +1,41 @@
-*{
-    font-family: Roboto;
-}
-h6{
-    margin-top: 14px;
-}
-.card{
-    margin: 0 auto; /* Added */
-    /* float: none; Added */
-    /* margin-bottom: 30px; Added */
-    border-radius: 15px;
-    height: auto;    
-}
\ No newline at end of file
+* {
+  font-family: Roboto;
+}
+
+.card.shadow {
+  width: 25rem;
+  margin-top: 80px;
+}
+.userlogo {
+  width: 60px;
+  height: 60px;
+}
+.fs-4 {
+  color: rgb(41, 41, 231);
+  font-weight: bold;
+}
+.form-control {
+  margin-top: 10px;
+  width: 100%;
+  border-radius: 1px;
+  box-shadow: none;
+  border: none;
+  border-bottom: 1px solid #d1d1d4 !important;
+}
+
+.card {
+  padding: 10px;
+  margin: 0 auto;
+  height: auto;
+}
+.d-grid {
+  margin-top: 20px;
+}
+.btn.btn-primary {
```

Comment:
again here too pelase remove this and use custom class instead or boostrpa class

Reply:
yes I have Remove bootstrap class in CSS and I have uses to custom class